README.md

```diff
@@ -1 +1,31 @@
-# hw01
\ No newline at end of file
+---
+title: "Yunshu Hu"
+---
+
+# hw01
+
+## Basic Info
+
+My name is Yunshu Hu, I am currently back home in Jinan, China. 
+
+I am a rising third year undergraduate student majoring in economics.
+
+Here is the most recent picture of me I can find in my phone. It is taken by my best friend.
+
+![A picture of me](Yunshu_pic.jpg)
```

Comment:
Just a note - while you have the text "A picture of me", it doesn't show up in the file (with the photo). I'm not sure if your intention was to have that showing, but just wanted to make you aware!

Reply:
Same here. I can see the picture but not the text "a picture of me."

Reply:
great photo!

Reply:
The caption text is used as a placeholder for people with slow internet or web browsers that cannot render images. It is a holdover from the old days of computers. This is all correct here!